README.md

```diff
@@ -1,27 +1,151 @@
 ###Ruby Wrapper to the PlotLy REST API
 
-**Example Usage**
+**Connecting to the API**
 
 ```ruby
 
 plotly = PlotLy.new('username', 'api key')
```

Comment:
Please add:
You can get your API key from https://plot.ly/settings/ when sign-into Plotly with a free account.